### ISSUE TITLE
Importing GPKG Cache

### ISSUE THREAD

**Issue by Gideon357** [domlysz/BlenderGIS#429]
# **Blender and OS versions**
Blender 2.83 and 2.9, Windows 10

# **Describe the bug**

**tl;dr:** I lost some files, but I have the cache. Is there a way to import the GPKG files from the cache folder?

This isn't a bug per se. It's more of a feature request / usage question. My Blender crashed while I was using the addon (not due to the addon, it was an unrelated issue), and I lost some of my files. I noticed that the addon caches GPKG files, which are GeoPackage data files. Is  it possible to import or convert these files so that they can be used with the addon? I assume something like converting them to GeoJSON or SHP files would do this. Will the addon ever support these? If not, is there another way to recover the lost files? If I can just get the coordinates, I can redo everything else. I think they may be in the logs.

**Comment by domlysz** [comment]
The geopackage file contains only basemap's tiles, you can open it with QGIS. Cached tiles are reused every time you run he basemap operator. There is no vector data inside it, these kind of data is stored in the *.blend file as native blender's mesh datablock.

**Comment by Gideon357** [comment]
> The geopackage file contains only basemap's tiles, you can open it with QGIS. Cached tiles are reused every time you run he basemap operator. There is no vector data inside it, these kind of data is stored in the *.blend file as native blender's mesh datablock.

So, in otherwords, my files are gone forever?

**Comment by domlysz** [comment]
I can tell you what the geopackage contains but I don't know what kind of file you lost, how this happen and what exactly you expect to found in the geopackage, so it's up to you to decide the conclusions.

**Comment by Gideon357** [comment]
> I can tell you what the geopackage contains but I don't know what kind of file you lost, how this happen and what exactly you expect to found in the geopackage, so it's up to you to decide the conclusions.

Ok. I lost a file that was just a 3D model from the addon. I forgot to write down the coords. If I can recover the coords, or the geo data, I can recreate everything.

**Comment by samstommy** [comment]
whats the point of the geopackage file if everytime i load the same sameare i need to redownload it?
